controls/stig_ctr/SRG-APP-000014-CTR-000035.yml

```diff
@@ -0,0 +1,9 @@
+controls:
+- id: SRG-APP-000014-CTR-000035
+  levels:
+  - medium
+  title: The container platform must use TLS 1.2 or greater for secure container image
```

Comment:
This won't hold up the PR, but replacing `The container platform` with `{{{ full_name }}}` might be helpful in the future.